### DISCUSSION TITLE
is it possible to use CF R2?

### DISCUSSION THREAD

**Discussion by ivanberry** [garethgeorge/backrest#609] [Q&A] [answered]
Note: if you have a question or want discussion please post in the [discussions area](https://github.com/garethgeorge/backrest/discussions).

**Is your feature request related to a problem? Please describe.**
is it possible to use CF R2 as the Repo

**Comment by garethgeorge** [comment, marked as answer]
Yes :) using restic and therefore backrest's rclone support

https://restic.readthedocs.io/en/latest/030_preparing_a_new_repo.html#other-services-via-rclone

https://developers.cloudflare.com/r2/examples/rclone/
